comment-dwim-2.el

```diff
@@ -60,8 +60,8 @@
 (require 'cl-lib)
 
 (defvar comment-dwim-2--inline-comment-behavior 'kill-comment
-  "Behavior of `comment-dwim-2' when it is being repeated and is
-encountering an inline comment.  Possible values are:
+  "Behavior of `comment-dwim-2' when repeated and at an inline comment.
```

Comment:
This variation is shorter - is it meaningfully different? I changed it in a similar way at a couple of points in this commit.